src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -17,6 +17,11 @@ public class AntiforgeryOptions
         private string _formFieldName = AntiforgeryTokenFieldName;
 
         /// <summary>
+        /// Represent the default cookie prefix, which is ".AspNetCore.Antiforgery".
+        /// </summary>
+        public static readonly string DefaultCookiePrefix = ".AspNetCore.Antiforgery";
+
+        /// <summary>
         /// Specifies the name of the cookie that is used by the antiforgery system.
         /// </summary>
         /// <remarks>
```

Comment:
Add that the automatically-generated name includes the `<see cref="DefaultCookiePrefix"/>` to these remarks.